core/data/src/main/java/com/google/samples/apps/nowinandroid/core/data/repository/DefaultSearchContentsRepository.kt

```diff
@@ -74,4 +74,13 @@ class DefaultSearchContentsRepository @Inject constructor(
             )
         }
     }
+
+    override fun getSearchContentsCount(): Flow<Int> {
```

Comment:
NIT: Simpler syntax can be used

`override fun getSearchContentsCount(): Flow<Int> = combine(...)`

Reply:
Done